MERN/Topic/Frontend/React/5.1_Connecting_with_API.md

```diff
@@ -0,0 +1,74 @@
+# **Getting started with API**
+
+## **How to connect with an API :**
+
+   <!-- <p> -->
+
+API stands for Application Program Interface, which can be defined as a set of methods of communication between various software components. In other words, an API allows software to communicate with another software.
+We can communicate and connect our frontend code with backend database using api.
+
+There are various method when calling and api. Those are </br>
+
+1.  ### _POST Method :_
+
+    It is used when someone try to add data to backend server. Ex : To add a new task to backend database we will use POST method.
+
+2.  ### _GET Method :_
+
+    It is used when you want to fetch some data from backend database . Ex : If you want to get a list of all task you have to use GET Method.
+
+3.  ### _PUT Method :_
+
+    It is used when you want to update some exisiting data in database. Ex : If you want to change task name in your database you have to use PUT Method.
+
+4.  ### _DELETE Method :_
+    It is used when you want to Delete some exisiting data from your database. Ex : If you want to remove any task from your database you have to use DELETE Method.
+
+## **How to use axios :**
```

Comment:
Also tell other alternates like `fetch` which can be used in the same place.

Reply:
They particularly mention to mention axios. Ok. I am mentioning fetch under ```  how to connect with API  ``` .